Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -15,40 +17,90 @@ public class TwineNode : MonoBehaviour {
 	[HideInInspector]
 	public string[] childrenNames;
 	public List<GameObject> parents = new List<GameObject> ();
+	public bool isDecisionNode;
 
 	private bool isMinimized = false;
+	private bool isOptionsGuiOpen = false;
+
+	private int selectedOptionIndex = 0;
 
 	void Update ()
 	{
-		if (Input.GetKeyDown (KeyCode.Tab))
-		{
-			this.ToggleMinimize ();
+		if (this.enabled) {
+			if (Input.GetKeyDown (KeyCode.Tab)) {
+				this.isMinimized = !this.isMinimized;
+			}
+
+			if (this.isDecisionNode) {
+				if (this.isOptionsGuiOpen && Input.GetKeyDown (KeyCode.Q)) {
+					// Press Q to scroll through the children nodes
+					this.selectedOptionIndex = (this.selectedOptionIndex + 1) % (children.Length);
+				} else if (this.isOptionsGuiOpen && (Input.GetKeyDown (KeyCode.KeypadEnter) || Input.GetKeyDown (KeyCode.Return))) {
+					// Press ENTER or RETURN to select that child
+					this.ActivateChildAtIndex (selectedOptionIndex);
+				} else if (this.isOptionsGuiOpen && Input.GetKeyDown (KeyCode.E)) {
+					// E closes the options list
+					this.isOptionsGuiOpen = false;
+				}
+
+				// Q opens the options list if it's not already open
+				if (!this.isOptionsGuiOpen && Input.GetKeyDown (KeyCode.Q)) {
+					this.isOptionsGuiOpen = true;
+				}
+			}
 		}
 	}
 
 	public void OnGUI()
 	{
 		if (this.enabled && !this.isMinimized) {
 
-			float frameWidth = Screen.width / 3;
-			Rect frame = new Rect (10, 10, frameWidth, Screen.height);
+			float frameWidth = Math.Min(Screen.width / 3, 350);
+			float frameHeight = Math.Min(Screen.height / 2, 500);
+			Rect frame = new Rect (10, 10, frameWidth, frameHeight);
 
 			GUI.BeginGroup (frame);
 			GUIStyle style = new GUIStyle (GUI.skin.box);
 			style.wordWrap = true;
 			style.fixedWidth = frameWidth;
 			GUILayout.Box (this.content, style);
+
+			if (isDecisionNode) {
+				GUIStyle decisionHintStyle = new GUIStyle (style);
+				decisionHintStyle.fontStyle = FontStyle.BoldAndItalic;
+
+				if (!isOptionsGuiOpen) {
+					GUILayout.Box ("Press Q to progress in the story...", decisionHintStyle);
+				} else {
+					GUILayout.Box ("Press Q to scroll, E to close, ENTER to choose", decisionHintStyle);
+				}
+			}
+
+			if (this.isOptionsGuiOpen) {
+				// Draw list of buttons for the possible children nodes to visit:
+				GUIStyle optionButtonStyle = new GUIStyle (GUI.skin.button);
+				optionButtonStyle.fontStyle = FontStyle.Italic;
+				optionButtonStyle.wordWrap = true;
+
+				// Set highlighted button to have green text (this state is called `onNormal`):
+				optionButtonStyle.onNormal.textColor = Color.white;
+				// Set non-highlighted buttons to have grayed out text (state is called `normal`)
+				optionButtonStyle.normal.textColor = Color.gray;
```

Comment:
Good change. LITERALLY CAN NOT believe Unity decided 'normal' and 'onNormal' were good names for these states.